experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -801,6 +773,38 @@ export const FormAudience = ({
               <FormErrors name="proposedDuration" />
             </InputGroup>
           </Form.Group>
+
+          {experiment.isRollout ? (
```

Comment:
This change also switches the order of "Experiment duration" and "Enrollment period", right? Can we keep the order the same (e.g. for non-rollouts, "Enrollment period" is on the left and "Experiment duration" is on the right):
<img width="2057" alt="image" src="https://github.com/mozilla/experimenter/assets/43795363/52d0546b-cc2b-4d36-acd3-b039f3954c65">

Reply:
(For context, if we change the order of these fields it will definitely confuse our users and they may enter incorrect values)

Reply:
I think we would actually like to have the field greyed out instead of totally hidden (see comment [here](https://github.com/mozilla/experimenter/issues/9027#issuecomment-1789616457)). Sorry to change the requirements on you mid-PR 😅

Reply:
Hi @eliserichards. No Worries. Thanks for letting me know. I'll make the change.